WebDriverAgentMac/WebDriverAgentLib/Utilities/AMSessionCapabilities.m

```diff
@@ -21,3 +21,4 @@
 NSString* const AM_APP_ENVIRONMENT_CAPABILITY = @"environment";
 NSString* const AM_SKIP_APP_KILL_CAPABILITY = @"skipAppKill";
 NSString* const AM_NO_RESET_CAPABILITY = @"noReset";
+NSString* const AM_APP_PATH_CAPABILITY = @"appPath";
```

Comment:
the new capability should be defined in https://github.com/appium/appium-mac2-driver/blob/master/lib/desired-caps.js and documented in README